src/commcare_cloud/commands/ansible/ops_tool.py

```diff
@@ -109,3 +110,36 @@ def run(self, args, manage_args):
             print(template.format(
                 list(stats['pooling'])[0], queue_name, workers, concurrency_, concurrency_ // workers
             ))
+
+
+class PillowResourceReport(CommandBase):
+    command = 'pillow-resource-report'
+    help = """
+    Report of pillow resources.
+    """
+
+    arguments = ()
+
+    def run(self, args, manage_args):
+        environment = get_environment(args.env_name)
+        pillows = environment.app_processes_config.pillows
+        by_process = defaultdict(lambda: {'num_processes': 0, 'total_processes': None})
+        for host, processes in pillows.items():
+            for name, options in processes.items():
+                queue = by_process[name]
+                queue['num_processes'] += options.get('num_processes', 1)
+                total_processes = options.get('total_processes', 1)
+                if queue['total_processes'] is None:
+                    queue['total_processes'] = total_processes
+                elif queue['total_processes'] != total_processes:
```

Comment:
Should this check be done once call hosts have been traversed?

Reply:
that would require keeping a list of all of them, not really sure this even belongs here